.appveyor.yml

```
@@ -0,0 +1,32 @@
+environment:
+  matrix:
+  - julia_version: 1
```

```suggestion
  - julia_version: 1.0
```
The way I understand the comment at the bottom [here](https://github.com/JuliaCI/Appveyor.jl#usage) is that `1` means "the latest version with a `1` in front" which is not what we want here.